lib/challenge.js

```diff
@@ -48,7 +66,14 @@ var pet_owner;
   // Every pet can:
   // - make noise (each pet makes its own unique noise, as specified by `noise`.
 
-var some_pet;
+var some_pet = {
+  name: "Spud",
+  species: "Dog",
+  breed: "Lab",
+  noise: = function () {
+    return "I'm a" + species + "I say Woof Woof.";
+}
+}
```

Comment:
Well done. That `=` that needs to be removed is something you could easily figure out by trying to run this code and looking closely at it, and your answer mostly satisfies the question. 3/4 credit.